workos/utils/auto_pagination.py

```diff
@@ -0,0 +1,112 @@
+import workos
+from workos.utils.pagination_order import Order
+
+
+def timestamp_compare(timestamp_0, timestamp_1):
+    if timestamp_0 < timestamp_1:
+        return Order.Asc
+    else:
+        return Order.Desc
+
+
+def get_response(type, after, order, directory=None):
```

Comment:
Similar to the comment above, the logic in the list and auto-pagination should be generic, and not reference any specific object type.

Reply:
Updated `directory` to `parent_resource_id`

Reply:
Can we remove all references to specific object types in the pagination logic? The idea here is that the list object and auto-pagination wouldn't reference anything specific in the object type - only the parameters needed to page through the list, which are standard across all object types (`after`, `before`, and `order`). It should apply any other query params, but for those specific to the object, they should be generically applied.

The idea behind this is if we add a new resource, or add an additional query parameter to an existing resource, we don't need to update the list object or the pagination methods. We would only need to update these if something changes in the list or pagination as a whole.

For example, in the Stripe Python SDK, the list object is purely generic, so it can be applied to any object type, but doesn't reference any specific object type: https://github.com/stripe/stripe-python/blob/master/stripe/api_resources/list_object.py